docs/howto/operate/ci-cd.md

```diff
@@ -0,0 +1,138 @@
+# Our CI/CD system
+
+The best place to look to learn about the latest state of our *automatic* hub deployment
```

Comment:
I think the first question we should ask is: what kind of documentation should this be, and when do we want people to use it? (is it a quick "how to" guide, or more like a reference explanation? I feel like this is a bit of a combination of both - there is some "do this, do that" programmatic stuff, and some "here's how this is structured" high level stuff. I think it'd be good if we kept those kinds of information separate (see https://diataxis.fr for a good framework there)

Reply:
That's was a good point! I re-ordered stuff in that direction.